INSTALL.md

```diff
@@ -19,7 +19,9 @@
 
 Required Software:
 * C/C++ Compiler
-* [DPC++ Compiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html)
+* [DPC++ Compiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html) if building with SYCL support
+* Python version 3.9 or higher
+* TBB library (repository contains script to download it)
```

Comment:
there is a tricky thing, that depends on tbb version we must use MKL_FPK_GPU_VERSION_LINE=2024.0. This var controls what layout of tbb will be used. If tbb < 2024.7 its not necessary to use this flag, if higher, its necessary.

Reply:
Questions in this regard:
* Isn't the latest version of TBB set at `2021.13.0`?
* If the build script sources the TBB script that sets the environment variables, would it be possible to retrieve the version from the `.pc` file? Under the default oneapi-basekit install, it should be under `/opt/intel/oneapi/tbb/latest/lib/pkgconfig/tbb.pc`.